easybuild/tools/filetools.py

```diff
@@ -1177,35 +1177,28 @@ def find_backup_name_candidate(src_file):
     return dst_file
 
 
-def back_up_file(src_file, backup_extension="", hidden=False):
+def back_up_file(src_file, backup_extension='', hidden=False):
     """
     Backs up a file appending a backup extension and a number to it (if there is already an existing backup). Returns
     the name of the backup
 
     :param src_file: file to be back up
-    :param backup_extension: optional extension of the backup
-    :param hidden: make a hidden (leading dot) backup
+    :param backup_extension: optional extension to use for the backup file
+    :param hidden: make backup hidden (leading dot in filename)
     """
+    fn_pref, fn_suff = '', ''
```

Comment:
I would use `fn_prefix` and `fn_suffix`. But that's a minor thing, everything else looks great.